### PR TITLE
Claudia dev py37 (library/functions.po file)

### PR DESCRIPTION
**Pull request by clacri**
This only includes a few functions translation, but I want to merge so that I then start to work on the 3.8 version instead.

### CODE REVIEW

File: library/functions.po [hunk 1]
```diff
@@ -337,16 +353,22 @@ msgid ""
 "object, it has to define an :meth:`__index__` method that returns an "
 "integer. Some examples:"
 msgstr ""
+"Convierte un número entero a una cadena binaria con prefijo “0b”. El "
```
Comment (humitos) [inline library/functions.po:356]:
```suggestion
"Convierte un número entero a una cadena binaria con prefijo \"0b\". El "
```

Aquí hay que usar las comillas normales, que se "escapan" con barra `\`

File: library/functions.po [hunk 2]
```diff
@@ -337,16 +353,22 @@ msgid ""
...
+"resultado es una expresión de Python válida. Si *x* no es un objeto de "
+"clase :class:`int`  en Python, tiene que definir un método :meth:"
+"`__index__`  que devuelva un entero. Algunos ejemplos:"
 
 #: ../Doc/library/functions.rst:94
 msgid ""
 "If prefix \"0b\" is desired or not, you can use either of the following ways."
 msgstr ""
+"Según se desee o no el prefijo “0b”, se puede usar uno u otro de las "
```
Comment (humitos) [inline library/functions.po:365]:
```suggestion
"Según se desee o no el prefijo \"0b\", se puede usar uno u otro de las "
```

File: library/functions.po [hunk 3]
```diff
@@ -357,11 +379,17 @@ msgid ""
 "It cannot be subclassed further.  Its only instances are ``False`` and "
 "``True`` (see :ref:`bltin-boolean-values`)."
 msgstr ""
+"Devuelve un booleano, es decir, o bien ``True`` o ``False``. *x* es "
+"convertido usando el estándar :ref:`truth testing procedure <truth>`. Si *x* "
+"es falso o omitido, devuelve ``False``; en caso contrario devuelve  "
```
Comment (humitos) [inline library/functions.po:384]:
```suggestion
"es falso u omitido, devuelve ``False``; en caso contrario devuelve  "
```

File: library/functions.po [hunk 4]
```diff
@@ -1,219 +1,224 @@
...
 msgstr ""
 "Project-Id-Version: Python 3.7\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2019-05-06 11:59-0400\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: python-doc-es (https://mail.python.org/mailman3/lists/docs-es.python.org)\n"
+"PO-Revision-Date: 2020-05-07 10:23+0200\n"
+"Language-Team: python-doc-es (https://mail.python.org/mailman3/lists/docs-es."
+"python.org)\n"
```
Comment (humitos) [inline library/functions.po:14]:
```suggestion
```

**Comment by clacri** [comment]
Muchas gracias, ya los he aceptado todos.

**Comment by humitos** [comment]
I'm going to merge this so @clacri can continue working on this file but in 3.8

**Pull request merged**